server/src/test/java/com/google/finapp/FinAppIT.java

```diff
@@ -54,13 +54,15 @@
 
 @Category(IntegrationTest.class)
 public class FinAppIT {
-
-  private SpannerDaoInterface spannerDao;
-  private DatabaseClient databaseClient;
   private static Database db;
+  private DatabaseClient databaseClient;
+  private FinAppService finAppService;
```

Comment:
nit: Is this variable used anywhere ? We could perhaps just rename the blockingstub below to finAppService.